.github/workflows/ci-build-push-e2e-tests-on-pr.yaml

```
@@ -0,0 +1,64 @@
+  build-push-e2e-image-on-pr:
+    name: Build and push E2E Test Image On PR
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_BUILDER: podman
+    steps:
+      - name: Get merge commit
+        uses: ./.github/workflows/get-merge-commit.yaml
+
+      - name: Checkout PR head
+        uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5.0.1
+        if: needs.get-merge-commit.outputs.mergedSha
```

actionlint (1.7.10)

property "get-merge-commit" is not defined in object type {}

(expression)